searchsummary/src/vespa/searchsummary/docsummary/docsumfieldwriter.h

```diff
@@ -46,20 +43,27 @@ class IDocsumFieldWriter
     static const vespalib::string _empty;
 };
 
+class IDocsumFW : public IDocsumFieldWriter
```

Comment:
It seems like this class is for those that do not care about GeneralResult* from the insertField() function. Consider giving it at more descriptive name and add a class comment. My suggestion is SimpleDocsumFieldWriter.

Reply:
Agreed, ISimpleDFW it is.